packages/azure-kusto-ingest/src/managedStreamingIngestClient.ts

```diff
@@ -83,12 +84,16 @@ class KustoManagedStreamingIngestClient extends AbstractKustoClient {
         this.defaultDatabase = this.streamingIngestClient.defaultDatabase;
     }
 
-    async ingestFromStream(stream: StreamDescriptor | Readable, ingestionProperties?: IngestionPropertiesInput): Promise<any> {
+    /**
+     * Use Readable for Node.js and ArrayBuffer for browser
```

Comment:
A comment is the best we have? can't we protect it on the type level?

Reply:
I tried to avoid creating too many files - but we can create here as well (like ingestClient.ts) two new files managedStreamingIngestClient.browser.ts and a base class file and move the logic to base class and then expose it differently, or we can move to accepting only descriptors and only split the descriptor files

Reply:
done